frontend/views/components/ProfileCard.vue

```diff
@@ -176,12 +176,15 @@ export default ({
       )
     },
     toggleTooltip () {
-      this.$refs.tooltip.toggle()
+      this.$refs.tooltip?.toggle()
```

Comment:
@taoeffect I actually was not able to reproduce this error you attached:

![image](https://github.com/user-attachments/assets/cc0bf6c2-3efe-401b-9d02-f6dfa0d9275f)

But looking at the error stack trace which points to `sendMessage()`, it appears that some actions there like `this.redirect(...)` causes the `Tooltip.vue` to be destroyed earlier than it gets to `this.toggleTooltip()`. That makes `this.$refs.tooltip` become `undefined`. So I made a fix for it here.